.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
You wanna make sure to have a `.gitignore` file in your repo's so that you don't check in your dependencies to git.